octopusdeploy/schema_amazon_web_services_openid_connect_account.go

```diff
@@ -53,7 +54,7 @@ func expandAmazonWebServicesOpenIDConnectAccount(d *schema.ResourceData) *accoun
 	}
 
 	if v, ok := d.GetOk("session_duration"); ok {
-		account.SessionDuration = v.(string)
+		account.SessionDuration = strconv.Itoa(v.(int))
```

Comment:
nit: to avoid conversion, do you think we  can change type of `session_duration` to `int` in go-ostopusdeploy?

Reply:
No, the API actually expects a string even though it wants a number so we will still need to convert it.